### DISCUSSION TITLE
WGDashboard update

### DISCUSSION THREAD

**Discussion by Bogo268** [tteck/Proxmox#3733] [Help Needed] [answered]
Hello,
Do you have any ideea how to update WGdashboard to 4.0.3 ? 
using commands from wgdashboard repo does not work.
Thank you !

**Comment by tteck** [comment, marked as answer]
Try

```
cd /etc/wgdashboard/src
./wgd.sh update
```

**Reply by Bogo268** [reply]
Thank you !

**Reply by tteck** [reply]
Moving forward, type `update` in the LXC console.

**Reply by Bogo268** [reply]
> Moving forward, type `update` in the LXC console.

Thank you !

There is a way to auto-start tunnel after reboot ? 

BTW, Thank you for your hard work ! 🙏🏼

**Reply by tteck** [reply]
Auto start 🤷🏻‍♂️ 
https://github.com/donaldzou/WGDashboard/discussions/380
